### PR TITLE
deployment: packet.net ansible scripts

### PR DESCRIPTION
**Pull request by grahamwhaley**
Add a set of Ansible scripts to make packet.net machine deployment
easier and repeatable for Kata CI machines.

### CODE REVIEW

**Comment by grahamwhaley** [comment]
I don't know if we have any Ansible gurus out there (maybe @raravena80 ? :-) ) - but, I apologise for my no-doubt dubious Ansible :-)
@chavafg - you might want to glance over this
@Pennyzct - I don't know how you deploy the ARM packet.net machines, but maybe these scripts will work for or can be extended/adapted for that role as well?

**Comment by chavafg** [comment]
maybe @cboylan can help us take a look as I know he works with Ansible on Zuul.

File: deployment/packet/README.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+```
+
+This will deploy a new instance on packet.net, and wait for it to finish booting.
+The script will print out the machine details. Save these for later, as you will require
+the public IP address the instance has been allocated. You can also find the IP address
+of the instance on the packet.net web interface.
+
+## Configuring an instance
+
+The instance now needs to be configured and have the correct pacakges and users installed
```
Comment (chavafg) [inline deployment/packet/README.md]: s/pacakges/packages/

Comment (grahamwhaley) [reply deployment/packet/README.md]: fixed.

**Comment by chavafg** [comment]
lgtm

**Comment by GabyCT** [comment]
lgtm

**Pull request merged**